benchmarks/common/commonBuilder.js

```diff
@@ -0,0 +1,44 @@
+const { validateNotNil } = require('validation-utils')
+const { BenchmarkBuilder } = require('photofinish')
+const getopts = require('getopts')
+
+const options = getopts(process.argv.slice(1), {
+  alias: {
+    preset: 'p'
+  },
+  default: {}
+})
+
+const PRESET = {
+  LOW: (builder) => {
+    return builder
+      .warmupCycles(1000)
+      .benchmarkCycles(100)
+  },
+
+  MEDIUM: (builder) => {
+    return builder
+      .warmupCycles(1000)
+      .benchmarkCycles(1000)
+  },
+
+  HIGH: (builder) => {
+    return builder
+      .warmupCycles(1000)
+      .benchmarkCycles(100000)
+  }
+}
+
+function getCommonBuilder () {
+  const presetId = options.preset || 'LOW'
```

Comment:
when i tested yesterday, it failed to pass the p parameter down to commonBuilder.js, is it fixed now?

Reply:
unfortunately yes, options.preset is still empty.

Reply:
Aha, it has to be different. Writing suggestions on the other remarks.